docs/operations/upgrade-gardener.md

```diff
@@ -0,0 +1,87 @@
+# Gardener Upgrade Guide
+
+Upgrading Gardener is a multi-step process that updates its core components, including the operator, control plane, `gardenlet`s, and the clusters it manages (seeds and shoots). To ensure a smooth and safe upgrade, you should follow a specific sequence of steps. This process works with Gardener's built-in reconciliation and rollout mechanisms, which automatically apply changes across your landscape.
+
+## 1. Prepare Before Touching Anything
+
+First, understand Gardener's [version skew policy](../deployment/version_skew_policy.md), which defines which component versions are compatible with each other.
+
+Next, carefully read the [release notes](https://github.com/gardener/gardener/releases) for your target version. The notes will detail any breaking changes, new features, and bug fixes.
```

Comment:
How about adding some hints on how to read / interpret the tags in the release notes? As far as I'm aware, there are `[DEPENDENCY]`, `[DEVELOPER]`, `[OPERATOR]` and `[USER]`. In the context of this guide, I would assume an operator would focus on `[OPERATOR]` and `[USER]`.